sqlcompyre/analysis/table_comparison.py

```diff
@@ -268,10 +269,10 @@ def column_matches(self) -> ColumnMatches:
         case_stmt = select(*cases).select_from(inner_join).subquery()
```

Comment:
I think the root cause is in this expression, but I'm not sure how to solve it here.